packages/core/src/types/handlers.ts

```diff
@@ -28,16 +29,9 @@ export type UserHandlers<T extends AnyHandlerEventTypes = EventTypes> = {
   onHover: Handler<'hover', check<T, 'hover'>>
 }
 
-export type ReactDOMAttributes = Omit<
-  React.DOMAttributes<EventTarget>,
-  'children' | 'dangerouslySetInnerHTML' | keyof UserHandlers
->
-
-type NativeHandlersKeys = keyof ReactDOMAttributes
+type NativeHandlersKeys = keyof DOMHandlers
```

Comment:
The `Omit` to prevent interference between the DOMHandlers and UserHandlers is missing.

```suggestion
type NativeHandlersKeys = keyof Omit<DOMHandlers, keyof UserHandlers>
```